### PR TITLE
Create sequence_empty

### PR DESCRIPTION
**Pull request by Fayad-Alman**
- Creates an empty tensor sequence 
- Input: shape, dtype (optional)
- Output: self.tensor

### CODE REVIEW

**Comment by Fayad-Alman** [comment]
Issue:
https://github.com/unifyai/ivy/issues/9158

**Comment by Fayad-Alman** [comment]
Hi @ZoeCD 
Just a friendly ping as a reminder to check the changes whenever you're ready :)

File: ivy/functional/backends/onnx_func/sequence_empty [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'''
+Creating an empty tensor sequence with given data type (default is float)
+'''
+
+from onnx import helper
+
+
+class SequenceEmpty:
```
Comment (ZoeCD) [inline ivy/functional/backends/onnx_func/sequence_empty:8]: Hi! Sorry for the lack of proper instructions for this task. What we need is the backend implementation for `SequenceEmpty`. We just want to add some basic functions that ONNX supports. You can follow [these instructions](https://lets-unify.ai/ivy/contributing/open_tasks.html#ivy-experimental-api) to see what we expect from the PR. Thank you for your interest in helping Ivy! 😊

**Review by ZoeCD — CHANGES_REQUESTED** [review]

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by Aarsh2001** [comment]
Hi @Fayad-Alman, apologies for the last moment changes but these functions are supposed to be added to the Experimental API. Kindly add it in the respective folders, and re-request a review thanks.

**Comment by Fayad-Alman** [comment]
@Aarsh2001 Noted! Thanks!

**Comment by Fayad-Alman** [comment]
#9915

**Comment by Fayad-Alman** [comment]
Hi @ZoeCD 
I created a new PR for the proper changes. Mind taking a look?
#9973

**Comment by ZoeCD** [comment]
I left a comment in the new PR. I'm closing this one

**Pull request closed without merging**